sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityResult.java

```diff
@@ -58,23 +59,29 @@
     private SensitivityComputationStatus status;
     private final StateResult nStateResult = new StateResult();
     private final Map<String, StateResult> contingencyResults = new HashMap<>();
+    private final Map<String, StateResult> contingencyResultsAfterCra = new HashMap<>();
 
     public SystematicSensitivityResult() {
         this.status = SensitivityComputationStatus.SUCCESS;
     }
 
     public SystematicSensitivityResult completeData(SensitivityAnalysisResult results) {
+        return completeData(results, false);
+    }
```

Comment:
i think having only the method where we explicitely need the afterCra argument would be safer than having both these methods

Reply:
ok